Failed to build libxml-ruby with native extensions

I can't install libxml-ruby with bundler. I'm using rails 3.2.6. 

`
valentin@Ganimed:~ $ uname -a
Linux Ganimed 3.2.0-27-generic #43-Ubuntu SMP Fri Jul 6 14:25:57 UTC 2012 x86_64 x86_64 x86_64 GNU/Linux
`

`
valentin@Ganimed:~ $ ruby -v
rubinius 2.0.0dev (1.9.3 a33eda29 yyyy-mm-dd JI) [x86_64-unknown-linux-gnu]
`

`
Installing libxml-ruby (2.3.3) with native extensions 
Gem::Installer::ExtensionBuildError: ERROR: Failed to build gem native extension.

```
    /home/valentin/.rvm/src/rbx-head/staging/bin/rbx extconf.rb 
```

checking for socket() in -lsocket... no
checking for gethostbyname() in -lnsl... yes
checking for atan() in -lm... no
checking for atan() in -lm... yes
checking for inflate() in -lz... yes
checking for iconv_open() in -liconv... no
checking for libiconv_open() in -liconv... no
checking for libiconv_open() in -llibiconv... no
checking for iconv_open() in -llibiconv... no
checking for iconv_open() in -lc... yes
checking for xmlParseDoc() in -lxml2... yes
checking for libxml/xmlversion.h... no
checking for libxml/xmlversion.h in /opt/include/libxml2,/usr/local/include/libxml2,/usr/include/libxml2... yes
creating extconf.h
creating Makefile

make
compiling ruby_xml_error.c
ruby_xml_error.c: In function ‘rxml_set_handler’:
ruby_xml_error.c:43:3: error: too many arguments to function ‘rb_cvar_set_internal’
/home/valentin/.rvm/src/rbx-head/staging/vm/capi/19/include/ruby/ruby.h:1101:11: note: declared here
make: **\* [ruby_xml_error.o] Error 1

Gem files will remain installed in /home/valentin/.rvm/gems/rbx-head/gems/libxml-ruby-2.3.3 for inspection.
Results logged to /home/valentin/.rvm/gems/rbx-head/gems/libxml-ruby-2.3.3/ext/libxml/gem_make.out
An error occured while installing libxml-ruby (2.3.3), and Bundler cannot continue.
Make sure that `gem install libxml-ruby -v '2.3.3'` succeeds before bundling.
`

--- a/vm/capi/19/include/ruby/intern.h
+++ b/vm/capi/19/include/ruby/intern.h
@@ -1,5 +1,3 @@
 /* Stub file provided for C extensions that expect it. All regular
  * defines and prototypes are in ruby.h
  */
-
-#define RB_CVAR_SET_4ARGS 1

